spyder/plugins/statusbar/plugin.py

```diff
@@ -176,3 +198,41 @@ def clock_status(self):
     def _statusbar(self):
         """Reference to main window status bar."""
         return self._main.statusBar()
+
+    def _organize_status_widgets(self):
+        """
+        Organize the status bar widgets once the application is loaded.
+        """
+        # Desired organization from
+        internal_layout = [
+            'clock_status', 'cpu_status', 'memory_status', 'read_write_status',
+            'eol_status', 'encoding_status', 'cursor_position_status',
+            'vcs_status', 'interpreter_status', 'lsp_status', 'kite_status']
+        external_left = list(self.EXTERNAL_LEFT_WIDGETS.keys())
+
+        # Remove all the widgets from the statusbar

```

Comment:
```suggestion
        # Remove all widgets from the statusbar, except the external right
        # widgets because they are already at the right position.
```

With this you can remove the comment below.